DOCS-13082 add backup and restore scenarios (#584)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/manage-backup-restore.adoc

> = Back Up and Restore Runtime Fabric

Reviewer: Isn't Back up just one word?

./rtfctl backup <path_to_backup_file>
----

> This command creates a backup of the current system state in `<path_to_backup_file>`, which can be any path in the file system that you have write access to, for example: `/opt/anypoint/runtimefabric/backup.tar.gz`. 

Reviewer 2: I think this should be "such as" instead of "for example"

> This command creates a backup of the current system state in `<path_to_backup_file>`, which can be any path in the file system that you have write access to, for example: `/opt/anypoint/runtimefabric/backup.tar.gz`. 

Author: ```suggestion
This command creates a backup of the current system state in `<path_to_backup_file>`, which can be any path in the file system that you have write access to, such as: `/opt/anypoint/runtimefabric/backup.tar.gz`. 
```
